components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -1363,6 +1328,18 @@ else if (getZCTCoords(plane)[2] < timestamps.size()) {
     segments = null;
   }
 
+  // -- ZeissCZI-specific methods --
+
+  public boolean allowAutostitching() {
+    return ((DefaultMetadataOptions) getMetadataOptions()).getBoolean(
```

Comment:
Is there anything you want me to actively change here at the moment, or is the next to-do to have a call to sort out API/inheritance changes?